mavlink_vehicles/mavlink_vehicles.hh

```diff
@@ -187,6 +187,11 @@ class mav_vehicle
     bool autocontinue_after_rotation = false;
     mission_status autocontinue_action = mission_status::NORMAL;
 
+    std::chrono::time_point<std::chrono::system_clock> stop_time =
+        std::chrono::system_clock::from_time_t(0);
+    int time_stopped_ms = -1;
```

Comment:
As said in [here](https://github.com/01org/mavlink-vehicles/pull/5#discussion-diff-78393869R1260)